app/agents_v2/primary_agent/prompts/response_formatter.py

```
@@ -0,0 +1,529 @@
+            if any(re.search(pattern, structure.has_closing) for pattern in closing_patterns):
+                closing_score = 1.0
+            else:
+                closing_score = 0.6
+                improvement_suggestions.append("Closing could be more supportive and inviting")
+        else:
+            improvement_suggestions.append("Missing supportive closing")
+        
+        section_scores[ResponseSection.SUPPORTIVE_CLOSING] = closing_score
+        
+        # Check markdown formatting
+        markdown_score = 1.0 if structure.has_proper_markdown else 0.3
```

Ruff (0.11.9)

Local variable `markdown_score` is assigned to but never used

Remove assignment to unused variable `markdown_score`

(F841)